src/components/Dashboard/PatientCard/index.js

```diff
@@ -121,6 +173,26 @@ const PatientCard = (props) => {
             &nbsp;<FieldValue>{patient.age}</FieldValue>
           </PatientGeneralInfo>
         ) : null}
+        <ButtonWrap className="ml-2 ">
+          <div
+            role="button"
+            onClick={() => {
+              exportVitals(patient.patientId);
+            }}
+            onKeyDown={() => {
+              exportVitals(patient.patientId);
+            }}
```

Comment:
shouldn't we check if the key that is pressed is `Enter`? here?

Reply:
also instead of `div` can we use `Button` like we have done on the other component?

Reply:
the onKeyDown checks if  the key that is pressed is Enter.
The other component requires button  in web view
In mobile view the excel logo is used instead to download, Hence div was use for it.